stack.yaml

```diff
@@ -131,3 +142,11 @@ ghc-options:
   cardano-sl:            -Werror
   cardano-sl-tools:      -Werror
   cardano-sl-lwallet:    -Werror
+  cardano-sl-explorer:   -Werror
+
+# Added for explorer
+flags:
+  cardano-sl:
+    with-explorer: True
```

Comment:
This is wrong. If we want to build explorer version by default, then we should change default value of `with-explorer` in `cardano-sl.cabal`. But I think we shouldn't build explorer version by default. It should be handled by a script which builds explorer.